tests/integ/modin/window/test_rolling.py

```diff
@@ -126,6 +126,22 @@ def test_rolling_series(window, min_periods, center, agg_func):
             )
 
 
+@pytest.mark.parametrize("ddof", [-1, 0, 0.5, 1])
```

Comment:
What happens when ddof > 1 such as 2? Would that be supported or only 0 / 1?

Reply:
Added a test for this, turned the IFF expression into a CASE_WHEN :)

Reply:
Nice, makes sense!